src/Symfony/Component/VarDumper/Test/VarDumperTestTrait.php

```diff
@@ -19,6 +19,23 @@
  */
 trait VarDumperTestTrait
 {
+    private static $casters = [];
```

Comment:
As these properties could be accessed by class using the trait but are not really meant to be used directly, perhaps we should mark it `@internal`. WDYT?